zotero2readwise/zotero.py

```diff
@@ -149,7 +149,9 @@ def get_item_metadata(self, annot: Dict) -> Dict:
             # "date": data["date"],
             "tags": data["tags"],
             "document_type": data["itemType"],
-            "source_url": top_item["links"]["alternate"]["href"]
+            "source_url": top_item["links"]["alternate"]["href"],
+            "creators": "",
```

Comment:
@jmhammond Why `creators` and `attachment_url` fields are created here? Is there any unexpected behaviour that this tries to solve?